templates/CRM/MembershipExtras/Page/NextPeriodTab.tpl

```diff
@@ -1,5 +1,4 @@
 <script>
-var recurringContributionID = {$recurringContributionID};
 var financialTypes = JSON.parse('{$financialTypes|@json_encode}');
```

Comment:
`currentFinancialTypes` from Current Period tab should still be available here. No need for this line.